src/ctelnet.cpp

```diff
@@ -2962,7 +2962,7 @@ void cTelnet::processSocketData(char* in_buffer, int amount)
         }
         // TODO: https://github.com/Mudlet/Mudlet/issues/5780 (4 of 7) - investigate switching from using `char[]` to `std::array<char>`
         buffer[static_cast<size_t>(datalen)] = '\0';
-        if (mpHost->mpConsole->mRecordReplay) {
+        if (!loopbackTesting && mpHost->mpConsole->mRecordReplay) {
```

Comment:
```suggestion
        if (Q_LIKELY(!loopbackTesting) && mpHost->mpConsole->mRecordReplay) {
```

Reply:
Given that that is an argument supplied to the method (and is defaulted) and thus is indeed likely to be the case - I'm not sure that this will lead to *much* optimisation - TBH it is more likely to just end up as a hint to others browsing the code. :grinning: